### PR TITLE
Add smart contract support, removed FRP, change website name, and fixed player's world

### PR DESCRIPTION
**Pull request by clxyder**
KIND-5 Add support smart contract support to repo
KIND-9 Make the player's world appear
KIND-10 Remove frp from the repo
KIND-11 Change "Our Worlds" to "ourworlds" for the website's name

### CODE REVIEW

**Review by colmogil — APPROVED** [review]

File: app/src/lib/contract.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
-import upgradeableAbi from '/src/token/upgradeable.json';
+import upgradableWorld from '/src/token/UpgradableWorld.json';
+import contractAddress from '/src/token/contract-address.json';
+
+export const upgradableAbi = upgradableWorld.abi;
 
 export const globeTextureUrl = "https://ipfs.io/ipfs/QmV3w8D47cCJgjtcYuJF1LjLW5zpYUWU5Y7M21tzSyYtPE?filename=texture.png";
 
-export const CONTRACT_ADDRESS = "0xF4f7124b560F1816aF25A47F4b0d91673C1CEf5C";
+export const CONTRACT_ADDRESS = contractAddress.address;
```
Comment (luisvillamil) [inline app/src/lib/contract.js:8]: Great find

File: app/src/routes/wizards/membership.svelte [hunk 2]
```diff
@@ -68,7 +65,7 @@
             const options = {
                 chain: "avalanche testnet",
                 address: provider?.User.current()?.get('ethAddress'),
-                token_address: "0x7cC8a5B05bAf9b39AFd2d115C7291fA33b3ecC01"
+                token_address: CONTRACT_ADDRESS
```
Comment (luisvillamil) [inline app/src/routes/wizards/membership.svelte:68]: good stuff

File: README.md [hunk 3]
```diff
@@ -4,19 +4,17 @@ Turning kind deeds into freedom.
...
-Clone repository and set the feature branch
+Clone repository
 
-```bash
-clone https://github.com/our-worlds/ourworlds-svelte.git
+`git clone https://github.com/our-worlds/ourworlds-svelte.git`
 
-cd .\ourworlds-svelte\
-```
+`cd .\ourworlds-svelte\`
```
Comment (colmogil) [inline README.md:11]: Ohhhhh yeah.

**Review by luisvillamil — APPROVED** [review]
DEVELOPERS

**Comment by clxyder** [comment]
Before we merge, I think I am going to look into how to set Git Flow and then send these commits to develop instead of main

**Pull request closed without merging**